runtime/onert/core/src/backend/controlflow/ExternalContext.h

```diff
@@ -36,10 +42,12 @@ namespace controlflow
 // TODO Unify this with cpu::ExternalContext
 class ExternalContext : public IExternalContext
 {
+#ifdef USE_RUY
```

Comment:
before offline talk, I thought that we use RUY only for calculation. In here, we use RUY for thread library, not calculation. If you comment that here, it would be easier to read the code.

Reply:
In here, that identifier(USE_RUY) prevents build failure by `ruy::Context` when that identifier was not defined. And the `ruy::Context` must be required for using RUY. I think it seem not to be related to code readability.